### PR TITLE
Getting Users from reviewer and replacing CSS to Scss

### PR DESCRIPTION
**Pull request by LiadAzulay**

### CODE REVIEW

File: projects/storyteller/webapp/src/app/app.component.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .container {
-    background-color: #eee;
+  background-color: #eee;
 }
+
```
Comment (webdevelopland) [inline projects/storyteller/webapp/src/app/app.component.scss]: I think two empty lines it's too much :)

File: projects/storyteller/webapp/src/app/components/feed/feed.component.scss [hunk 2]
```diff
@@ -1,70 +1,4 @@
+@import 'app/styles/feeds.scss';
```
Comment (webdevelopland) [inline projects/storyteller/webapp/src/app/components/feed/feed.component.scss]: Wow, it's absolute way to import styles! Cool!

File: projects/storyteller/webapp/src/app/styles/feeds.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
+$font-stack: OpenSans;
+%header {
```
Comment (webdevelopland) [inline projects/storyteller/webapp/src/app/styles/feeds.scss]: Why not `.header`? I'm not saying it's an error, I just didn't see that syntax before

File: projects/storyteller/webapp/src/app/components/single-screenshot/single-screenshot.component.ts [hunk 4]
```diff
@@ -1,9 +1,59 @@
...
+    zip(
+      this.firebaseService.getUserStories(this.author),
+      this.firebaseService.getUserScreenshot(this.author)
+    ).subscribe(data => {
+      this.getStory(this.storyId, data[0]);
+      this.getScreenshot(this.screenshotFilename, data[1]);
+      this.isLoading = false;
+    });
+  }
+  getStory(storyId: string, storyLists: StoryList[]): void {
```
Comment (webdevelopland) [inline projects/storyteller/webapp/src/app/components/single-screenshot/single-screenshot.component.ts]: It'd be great if you'll put empty line between methods

File: projects/storyteller/webapp/src/app/core/services/firebase.service.ts [hunk 5]
```diff
@@ -25,13 +25,12 @@ export class FirebaseService {
...
     });
-    this.stories =
-      this.db.collection(`/storyteller/data/user/valerii.fedorenko.ua@gmail.com/story`);
-    this.screenshots =
-      this.db.collection('/storyteller/data/user/valerii.fedorenko.ua@gmail.com/screenshot');
+    this.protobin_users = this.db.collection('reviewer');
   }
-  getStorylistAll(): Observable<StoryList[]> {
-    return this.stories.snapshotChanges().pipe(
+  getstorylistAll(
```
Comment (webdevelopland) [inline projects/storyteller/webapp/src/app/core/services/firebase.service.ts]: camel case

File: projects/storyteller/webapp/src/app/core/services/firebase.service.ts [hunk 6]
```diff
@@ -25,13 +25,12 @@ export class FirebaseService {
     this.angularFireAuth.authState.subscribe(userData => {
       this.isOnline = !!userData;
     });
-    this.stories =
-      this.db.collection(`/storyteller/data/user/valerii.fedorenko.ua@gmail.com/story`);
-    this.screenshots =
-      this.db.collection('/storyteller/data/user/valerii.fedorenko.ua@gmail.com/screenshot');
+    this.protobin_users = this.db.collection('reviewer');
```
Comment (webdevelopland) [inline projects/storyteller/webapp/src/app/core/services/firebase.service.ts]: `protobin_users` -> `reviewerConfig`

File: projects/storyteller/webapp/src/app/core/services/firebase.service.ts [hunk 7]
```diff
@@ -1,21 +1,21 @@
...
 import { AngularFireAuth } from 'angularfire2/auth';
 import * as firebase from 'firebase/app';
+
 interface FirebaseElement {
   proto: string;
 }
 
 @Injectable()
 export class FirebaseService {
+
```
Comment (webdevelopland) [inline projects/storyteller/webapp/src/app/core/services/firebase.service.ts]: It's just question of style guide. Could you remove empty line here?

File: projects/storyteller/webapp/src/app/app.component.scss (hunk 1 above)
Comment (LiadAzulay) [reply projects/storyteller/webapp/src/app/app.component.scss]: Done

File: projects/storyteller/webapp/src/app/components/feed/feed.component.scss (hunk 2 above)
Comment (LiadAzulay) [reply projects/storyteller/webapp/src/app/components/feed/feed.component.scss]: Thanks vadim!! I wanted to ask  if I do it right...😀😀

File: projects/storyteller/webapp/src/app/styles/feeds.scss (hunk 3 above)
Comment (LiadAzulay) [reply projects/storyteller/webapp/src/app/styles/feeds.scss]: but if it's .header it's not working with inheritance in scss
https://github.com/hasadna/hasadna/pull/140/files/65f3fff4dbb7e3eae5f0c430ffd4f6d5dd8ffe46#diff-4dc0243ca547a4e54f5a6fd411ffd923R49

File: projects/storyteller/webapp/src/app/components/single-screenshot/single-screenshot.component.ts (hunk 4 above)
Comment (LiadAzulay) [reply projects/storyteller/webapp/src/app/components/single-screenshot/single-screenshot.component.ts]: Done

File: projects/storyteller/webapp/src/app/core/services/firebase.service.ts (hunk 5 above)
Comment (LiadAzulay) [reply projects/storyteller/webapp/src/app/core/services/firebase.service.ts]: done

File: projects/storyteller/webapp/src/app/core/services/firebase.service.ts (hunk 6 above)
Comment (LiadAzulay) [reply projects/storyteller/webapp/src/app/core/services/firebase.service.ts]: done

File: projects/storyteller/webapp/src/app/core/services/firebase.service.ts (hunk 7 above)
Comment (LiadAzulay) [reply projects/storyteller/webapp/src/app/core/services/firebase.service.ts]: done

File: projects/storyteller/webapp/src/app/components/feed/feed.component.ts [hunk 8]
```diff
@@ -27,23 +27,26 @@ export class FeedComponent {
...
       this.screenshots.push.apply(this.screenshots, data[1]);
-
       this.isLoading = false;
     });
   }
+  
   loadusers(): void {
     this.firebaseService.getReviewerConfig().subscribe(userData => {
       this.users = userData.getUserList();
+      console.log(this.users);
```
Comment (webdevelopland) [inline projects/storyteller/webapp/src/app/components/feed/feed.component.ts]: console.log :frowning_face:

File: projects/storyteller/webapp/src/app/styles/feeds.scss (hunk 3 above)
Comment (webdevelopland) [reply projects/storyteller/webapp/src/app/styles/feeds.scss]: What do you mean?

File: projects/storyteller/webapp/src/app/components/single-screenshot/single-screenshot.component.ts (hunk 4 above)
Comment (webdevelopland) [reply projects/storyteller/webapp/src/app/components/single-screenshot/single-screenshot.component.ts]: Great! But actually not only here. There's `FirebaseService` for example

File: projects/storyteller/webapp/src/app/core/services/firebase.service.ts (hunk 7 above)
Comment (webdevelopland) [reply projects/storyteller/webapp/src/app/core/services/firebase.service.ts]: Are you sure it's done?

File: projects/storyteller/webapp/src/app/app.component.scss (hunk 1 above)
Comment (webdevelopland) [reply projects/storyteller/webapp/src/app/app.component.scss]: Could you also find all other places with two empty lines?

**Pull request merged**